Pod/Classes/VOKCoreDataManager.m

```diff
@@ -319,46 +296,20 @@ - (NSArray *)importArray:(NSArray *)inputArray forClass:(Class)objectClass withC
         } else {
             returnObject = [self managedObjectOfClass:objectClass inContext:contextOrNil];
             [mapper setInformationFromDictionary:inputDict forManagedObject:returnObject];
+            if (inputKey) {
+                [existingObjectArray addObject:returnObject];
+                [existingUniqueKeys addObject:inputKey];
```

Comment:
oh, wait, never mind.  

This is confusing, perhaps adding some comments would help?